muselog/__init__.py

```diff
@@ -40,6 +40,11 @@ def setup_logging(root_log_level: Optional[str] = None,
         console_handler.setFormatter(logging.Formatter(fmt=console_handler_format or DEFAULT_LOG_FORMAT))
         root_logger.addHandler(console_handler)
 
+        # log to docker for datadog if enabled. 
+        if "ENABLE_DD_DOCKER_LOG" in os.environ and os.environ["ENABLE_DD_DOCKER_LOG"] == "True":
+            formatter = DatadogJSONFormatter()
+            console_handler.setFormatter(formatter)
```

Comment:
How about `ENABLE_DATADOG_JSON_FORMATTER`? muselog should have no idea about docker.

Reply:
Cool